packages/sanity/src/core/studio/tasks/components/TasksSidebar/TasksSidebar.tsx

```diff
@@ -0,0 +1,36 @@
+import {Card} from '@sanity/ui'
+import styled from 'styled-components'
+import {AnimatePresence, motion} from 'framer-motion'
+import {useTasksEnabled, useTasks} from '../../context'
+import {TasksSidebarHeader} from './TasksSidebarHeader'
+
+const SidebarRoot = styled(Card)`
+  height: 100%;
+  width: 360px;
+  margin-left: 4px;
```

Comment:
Can we use the `marginLeft` prop instead – so that we rely on theme values?

Reply:
Absolutely! Good catch.